packages/server/tests/ListUserFollowers.test.ts

```diff
@@ -0,0 +1,385 @@
+                searchQuery,
+                userID: searchedUserID,
+                tmpAuthUserID: requestingUserID,
+            } as ListUserFollowersRequestBody)
+            .expect('Content-Type', /json/)
+            .expect(200);
+
+        const pageBodyParsed = ListUserFollowersResponseBody.parse(pageBodyRaw);
+
+        assert.equal(pageBodyParsed.page, 1);
+        assert.equal(
+            pageBodyParsed.totalEntries,
+            matchingSearchQuerySortedByNicknameSearchedUserFollowers.length,
+        );
+        assert.isFalse(pageBodyParsed.hasMore);
+        assert.equal(
+            pageBodyParsed.data.length,
+            matchingSearchQuerySortedByNicknameSearchedUserFollowers.length,
+        );
+        const expectedData =
+            matchingSearchQuerySortedByNicknameSearchedUserFollowers.slice(
+                0,
+                PAGE_MAX_LENGTH,
+            );
+        console.log('ACTUAL', pageBodyParsed.data);
+        console.log('EXPECTED', expectedData);
+        assert.deepEqual(pageBodyParsed.data, expectedData);
+    });
+
+    test('It should send back 404 as requesting user does not exist', async () => {
+        const requestingUserID = datatype.uuid();
+        const searchedUserID = datatype.uuid();
+        await User.create({
+            uuid: searchedUserID,
+            nickname: internet.userName(),
+        });
+
+        await supertest(BASE_URL)
+            .post(urlcat(TEST_USER_ROUTES_GROUP_PREFIX, 'search/followers'))
+            .send({
+                page: 1,
+                searchQuery: '',
+                userID: searchedUserID,
+                tmpAuthUserID: requestingUserID,
+            } as ListUserFollowersRequestBody)
+            .expect(404);
+    });
+
+    test('It should send back 404 as searched user does not exist', async () => {
+        const requestingUserID = datatype.uuid();
+        const searchedUserID = datatype.uuid();
+        await User.create({
+            uuid: requestingUserID,
+            nickname: internet.userName(),
+        });
+
+        await supertest(BASE_URL)
+            .post(urlcat(TEST_USER_ROUTES_GROUP_PREFIX, 'search/followers'))
+            .send({
+                page: 1,
+                searchQuery: '',
+                userID: searchedUserID,
+                tmpAuthUserID: requestingUserID,
+            } as ListUserFollowersRequestBody)
+            .expect(404);
+    });
+
+    test('It should send back forbidden as searched user relations visibility is PRIVATE', async () => {
+        const requestingUserID = datatype.uuid();
+        const searchedUserID = datatype.uuid();
+        await User.create({
+            uuid: requestingUserID,
+            nickname: internet.userName(),
+        });
+        const searchedUser = await User.create({
+            uuid: searchedUserID,
+            nickname: internet.userName(),
+        });
+
+        const privateVisibility = await getVisibilityDatabaseEntry('PRIVATE');
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why did not you use `UserSettingVisibility.enum.PRIVATE` here?

Reply:
Oh forgot to comment on the PR about it
For your beautiful eyes of course ;) jking
Wanted to try it out